skeleton/frontend/frontend.cabal

```diff
@@ -24,6 +26,8 @@ executable frontend
                , common
                , obelisk-frontend
                , obelisk-route
+               , ghcjs-dom-jsffi
+               , jsaddle
```

Comment:
Shouldn't need them in the executable. In fact ... we probably can prune these even further...